### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by mugendiz**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <meta name="google-signin-client_id" content="558027906323-pp4gp3279cmjjulkggqn77vgklp58vhm.apps.googleusercontent.com">
```
Comment (mjsimon91) [inline client/public/index.html]: Should we be exposing this? Or should this be in a .env file?

File: client/src/App.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
 import React, { Component } from "react";
-import Navbar from "./components/Navbar"
+import ReactDOM from "react-dom";
```
Comment (mjsimon91) [inline client/src/App.js]: What is this being used for?

File: client/src/App.js [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
+
+import Navbar from "./components/Navbar";
 import Jumbotron from "./components/Jumbotron";
 import PodcastList from "./components/PodcastList";
 import "./App.css";
 
+const app = document.getElementById('app');
```
Comment (mjsimon91) [inline client/src/App.js]: What does this do?

File: client/src/components/GoogleLoginHelper/GoogleLoginHelper.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import GoogleLogin from 'react-google-login';
+
+class GoogleLoginHelper extends Component {
+  // ReactDOM.render
+  clientId = '558027906323-pp4gp3279cmjjulkggqn77vgklp58vhm.apps.googleusercontent.com'
```
Comment (mjsimon91) [inline client/src/components/GoogleLoginHelper/GoogleLoginHelper.js]: Again, Should the client id be hidden?

File: client/src/components/GoogleLoginHelper/GoogleLoginHelper.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  logout = () => {
+    console.log('logout')
+  }
+
+render() {
+  return(
+    <GoogleLogin
+    clientId="658977310896-knrl3gka66fldh83dao2rhgbblmd4un9.apps.googleusercontent.com"
```
Comment (mjsimon91) [inline client/src/components/GoogleLoginHelper/GoogleLoginHelper.js]: Should the id be exposed?

File: client/src/components/Navbar/Navbar.js [hunk 6]
```diff
@@ -1,15 +1,36 @@
...
+import GoogleLoginHelper from 'react-google-login';
+// import GoogleLoginHelper from "./components/GoogleLoginHelper";
+
+
+import { Router, Route, IndexRoute, hashHistory } from 'react-router';
+import App from '../../App.js';
+import SignIn from './SignIn';
 import './Navbar.css';
 
+const app = document.getElementById('app');
```
Comment (mjsimon91) [inline client/src/components/Navbar/Navbar.js]: What does this do

File: client/src/components/Navbar/SignIn.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (mjsimon91) [inline client/src/components/Navbar/SignIn.js]: What do we use this for?

Comment (mugendiz) [reply client/src/components/Navbar/SignIn.js]: we dont need to render to the DOM here

File: client/src/components/GoogleLoginHelper/GoogleLoginHelper.js (hunk 5 above)
Comment (mugendiz) [reply client/src/components/GoogleLoginHelper/GoogleLoginHelper.js]: ID can be hidden or the line bypassed

File: client/src/App.js (hunk 2 above)
Comment (mugendiz) [reply client/src/App.js]: I think that's incase we have reactDOM render

File: client/public/index.html (hunk 1 above)
Comment (mugendiz) [reply client/public/index.html]: couldn find a conclusive answer.. still looking

**Pull request closed without merging**